h/views/accounts.py

```diff
@@ -341,16 +350,17 @@ def get_when_not_logged_in(self):
             jinja2.Markup(
                 _(
                     "Your account has been activated! "
-                    'You can now <a href="{url}">log in</a> using the password you '
-                    "provided."
-                ).format(url=self.request.route_url("login"))
+                    "You can now log in using the password you provided."
+                ),
```

Comment:
This flash message is now shown on the login page so it doesn't need to link to the login page